From 6485c671d41b98471133dc1e78af6ddb8596e495 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Sat, 7 Jan 2017 17:32:36 -0800
Subject: [PATCH] Remove CM Stats feature

The URL is gone now and this feature is not relevant to unofficial builds.

Change-Id: Icbbad3e9f1ca45eb910640237b3515aff3471ade
---
 src/com/android/settings/cmstats/StatsUploadJobService.java | 1 +
 src/com/android/settings/cyanogenmod/PrivacySettings.java   | 5 +----
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/src/com/android/settings/cmstats/StatsUploadJobService.java b/src/com/android/settings/cmstats/StatsUploadJobService.java
index 580a20f..1b1ce37 100644
--- a/src/com/android/settings/cmstats/StatsUploadJobService.java
+++ b/src/com/android/settings/cmstats/StatsUploadJobService.java
@@ -99,6 +99,7 @@ public class StatsUploadJobService extends JobService {
         @Override
         protected Boolean doInBackground(Void... params) {
 
+            if (true) return true;
             PersistableBundle extras = mJobParams.getExtras();
 
             String deviceId = extras.getString(KEY_UNIQUE_ID);
diff --git a/src/com/android/settings/cyanogenmod/PrivacySettings.java b/src/com/android/settings/cyanogenmod/PrivacySettings.java
index f6a9c15..daa23e0 100644
--- a/src/com/android/settings/cyanogenmod/PrivacySettings.java
+++ b/src/com/android/settings/cyanogenmod/PrivacySettings.java
@@ -47,10 +47,7 @@ public class PrivacySettings extends SettingsPreferenceFragment {
         PackageManager pm = getPackageManager();
 
         boolean isOwner = Utils.isUserOwner();
-        if (!isOwner) {
-            PreferenceScreen root = getPreferenceScreen();
-            root.removePreference(findPreference(KEY_STATS));
-        }
+        getPreferenceScreen().removePreference(findPreference(KEY_STATS));
 
         // Determine options based on device telephony support
         if (!pm.hasSystemFeature(PackageManager.FEATURE_TELEPHONY) || !isOwner) {
-- 
2.9.3

